docs/_tutorials/messagevpn-with-queue-java.md

```diff
@@ -0,0 +1,246 @@
+
+To understand how to implement this, let’s consult now the [SEMP online API
+documentation]({{ site.docs-api }}){:target="_top"}:
+- scrolling down to msgVPN and opening the `List Operations` link will show the color-coded options for all available operations. Clicking on `POST /msgVpns - Creates a Message VPN object` will open up the details. Scrolling down to `MsgVpn {`, here we can find the names of the attributes and we can derive the setter/getter Java method names. For example we look up the attribute `MaxMsgSpoolUsage` and derive the setter method to be `setMaxMsgSpoolUsage(int)`.
+
+```java
+// Create message-vpn
+MsgVpn msgVpn = new MsgVpn();
+msgVpn.setMsgVpnName(messageVpnName);
+msgVpn.setAuthenticationBasicType(AuthenticationBasicTypeEnum.INTERNAL);
+msgVpn.setMaxMsgSpoolUsage(1500);
+msgVpn.setEnabled(true);
+MsgVpnResponse vpnResp = sempApiInstance.createMsgVpn(msgVpn, null);        
+```
+
+When a new message-VPN management object is created on the Solace message router, a new `default` Client Username object will also be created automatically, associated with a new `default` Client Profile and a new `default` ACL Profile object. We will use these created default objects as a starting point for further configuration as they require fewer properties to modify. Alternatively, it is possible to create new non-default objects if a more complex structure is required.
+
+The `default` ACL Profile doesn’t define any access restrictions so there is no need to modify that.
+
+### Modify the default Client Profile
+
+However, the `default` Client Profile needs to be modified to enable support for guaranteed messaging:
+
+```java
+// Modify existing default client-profile
+String DEFAULT_CLIENTPROFILE = "default";
+MsgVpnClientProfile clientProfile = new MsgVpnClientProfile();
+clientProfile.setAllowGuaranteedMsgSendEnabled(true);
+clientProfile.setAllowGuaranteedMsgReceiveEnabled(true);
+clientProfile.setAllowGuaranteedEndpointCreateEnabled(true);
+MsgVpnClientProfileResponse cpResp = sempApiInstance.updateMsgVpnClientProfile(messageVpnName,
+ DEFAULT_CLIENTPROFILE, clientProfile, null);
+```
+
+### Create new or modify the default Client Username
+
+The Client Username needs to get a password property assigned and enabled.  If we opt for using a username other than `default`, we need to create a new Client Username. Note that unless a different Client Profile and ACL profile is assigned, even a newly created Client Username will be associated with the default profiles.
```

Comment:
Be more accurate here. The "default" username means that any username that is not found will be authenticated with this username. It does not imply a user name "default" as most people believe initially.